changelog.d/13346.misc

```diff
@@ -0,0 +1 @@
+Fix long-standing bug where Synapse sometimes asked many different remote hosts for the same event.
```

Comment:
```suggestion
Fix long-standing bug in `get_pdu` asking every remote destination even after it finds an event.
```

Reply:
Ah, if its never triggered I wonder if we should say that? e.g. "..., which was never hit.", or something?